policycoreutils: fix genhomedircon generation

The script is put on the target and needs to reference
the directory in which semodule is installed.

Upstream Status: pending

Signed-off-by: Joe Slater <jslater@windriver.com>

--- a/scripts/Makefile
+++ b/scripts/Makefile
@@ -8,11 +8,12 @@ LOCALEDIR ?= $(PREFIX)/share/locale
 
 all: fixfiles genhomedircon chcat
 
+# we want the script to use the user sbin directory on the target
+#
 genhomedircon:
 	@echo "#!/bin/sh" > genhomedircon
 	@echo >> genhomedircon
-	@if [ ! -e semodule_path ]; then echo -n "$(USRSBINDIR)/" >> genhomedircon; fi
-	@if [ -e semodule_path ]; then echo -n "`cat semodule_path`/" >> genhomedircon; fi
+	@echo -n "$(sbindir)/" >> genhomedircon
 	@echo "semodule -Bn" >> genhomedircon
 
 install: all
